docs/debian-conf.d/transport/30_vexim_ditch_spam_transport

```diff
@@ -0,0 +1,20 @@
+
+### transport/30_vexim_ditch_spam_transport
+#######################################
+
+ditch_spam_transport:
```

Comment:
I'm not sure..the normal "ditch_spam" blackhole router doesn't have that, too. Here's the Problem we have with "no naming rules". Still I'm adding the virtual here.